lib/components/home/artwork_rails/artwork_rail_header.js

```diff
@@ -32,7 +32,13 @@ class ArtworkRailHeader extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = { following: false }
+
+    let following = false
+    if (props.rail.key === 'followed_artist') {
+      following = true
+    }
+
+    this.state = { following: following }
```

Comment:
Nitpick: this could’ve been just `this.state = { following: props.rail.key === 'followed_artist' }` no?